cmd/probe/udevprobe.go

```diff
@@ -146,6 +146,19 @@ func (up *udevProbe) scan() error {
 			deviceDetails.ProbeIdentifiers.Uuid = uuid
 			deviceDetails.ProbeIdentifiers.UdevIdentifier = newUdevice.GetSyspath()
 			diskInfo = append(diskInfo, deviceDetails)
+		} else if newUdevice.IsParitition() {
```

Comment:
Fold it in single if block something like this "if (newUdevice.IsDisk() || newUdevice.IsPartition()"
because we are doing same thing in both blocks difference is just getting partition info if this is partition which we can anyway do by checking if this is partiton.